robot_skills/src/robot_skills/api.py

```diff
@@ -42,10 +42,12 @@ def query(self, description, grammar, target, timeout=10):
         except TimeoutException as e:
             if callable(self._post_hook):
                 self._post_hook()
+            self.restart_dragonfly()
```

Comment:
So we want to restart dragonfly each time we have a time-out exception? And also after a succesfull query? So at al times?